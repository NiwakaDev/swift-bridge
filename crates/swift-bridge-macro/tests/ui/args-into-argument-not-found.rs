//! # To Run
//! cargo test -p swift-bridge-macro -- ui trybuild=args-into-argument-not-found.rs

#[swift_bridge::bridge]
mod ffi {
    extern "Rust" {
        #[swift_bridge(args_into = (arg, arg_typo))]
        fn some_function(arg: u8);
    }
}

fn some_function(arg: u8){

}

fn main() {

}