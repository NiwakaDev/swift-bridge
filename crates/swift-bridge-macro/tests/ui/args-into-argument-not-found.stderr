error: arg_typo: Unknown argument
       Could not find "arg_typo" in "fn some_function(arg : u8)".
 --> tests/ui/args-into-argument-not-found.rs:7:42
  |
7 |         #[swift_bridge(args_into = (arg, arg_typo))]
  |                                          ^^^^^^^^
